domain/credential/state/state_test.go

```diff
@@ -348,16 +340,30 @@ func (s *credentialSuite) TestRemoveCredentials(c *gc.C) {
 			"bar": "bar val",
 		},
 	}
-	key := corecredential.Key{Cloud: "stratus", Owner: s.userName, Name: "bobcred1"}
+	key := corecredential.Key{
+		Cloud: "foo",
+		Owner: usertesting.GenNewName(c, "test-userfoo"),
+		Name:  "foobar",
+	}
 	ctx := context.Background()
-	_, err := st.UpsertCloudCredential(ctx, key, cred1Info)
+	err := st.UpsertCloudCredential(ctx, key, cred1Info)
 	c.Assert(err, jc.ErrorIsNil)
 
+	models, err := st.ModelsUsingCloudCredential(ctx, key)
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(models, jc.DeepEquals, map[coremodel.UUID]string{
+		modelUUID: "foo",
```

Comment:
Wouldn't this be equal to modelUUID above?

Reply:
modelUUID is the key value
"foo" is the model name